core/src/main/scala/org/apache/gearpump/transport/netty/Context.scala

```diff
@@ -63,9 +63,9 @@ import org.apache.gearpump.transport.netty.Context._
   }
 
 
-  def bind(name: String, lookupActor : ActorLookupById): Int = {
+  def bind(name: String, lookupActor : ActorLookupById, deserializeFlag : Boolean = true): Int = {
     val taskDispatcher = system.settings.config.getString("gearpump.task-dispatcher")
```

Comment:
good point!

I now add a TODO comment in the source code.
I am not sure whether we want to allow a application to override this config for now.